source/docs/software/commandbased/convenience-features.rst

```diff
@@ -366,10 +366,10 @@ decorated command will be interrupted if the timeout expires:
     // Will time out 5 seconds after being scheduled, and be interrupted
     button.WhenPressed(command.WithTimeout(5.0_s));
 
-interruptOn
+withInterrupt
 ^^^^^^^^^^^
```

Comment:
```suggestion
withInterrupt
^^^^^^^^^^^^^
```

Reply:
What does the number of '^' mean?

Reply:
From https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections:
An underline/overline is a single repeated punctuation character that begins in column 1 and forms a line extending at least as far as the right edge of the title text.